validator_client/src/lib.rs

```diff
@@ -325,7 +325,7 @@ impl<E: EthSpec> ProductionValidatorClient<E> {
                     get_validator_block: slot_duration / HTTP_GET_VALIDATOR_BLOCK_TIMEOUT_QUOTIENT,
                 }
             } else {
-                Timeouts::set_all(slot_duration)
+                Timeouts::set_all(25 * slot_duration)
```

Comment:
@macladson could you make a clean PR for a CLI flag accomplishing the same effect against the `release-v7.0.0` branch? :pray:

Reply:
PR is now here: https://github.com/sigp/lighthouse/pull/7047